### ISSUE TITLE
Caching in dashboard

### ISSUE THREAD

**Issue by robertphipps** [pagekit/pagekit#684]
## Problem

When performing some tasks in the back-end I have to shift-reload in order for it to update - for example deleting a page or uploading a file. This means that you can upload a file, have the site claim that the upload is done but still not see the file, or delete a page, see the confirmation message but have the page still in the page list.

It turns out if I have the Chrome f12 developer tools window open the issue does not occur, I think that is because that disables caching?

There is also a similar issue where changes to the navbar don't necessarily update without a shift-reload on the front-end of the site, but this is much less of an issue.

## Possible solution

I haven't had a chance to look through the codebase, but do you have any kind of caching rules for logged in users or for the admin panel that send the `Cache-Control: no-cache` headers? I am assuming that this would fix the issue, but I may well be wrong.

Cheers,
Rob

## Technical Details

- Pagekit version: 1.0.3
- Webserver: Apache (on Cloudlinux cPanel)
- Database: MySQL 5.6.29
- PHP Version: Litespeed 5.6.18
- Browser: Chrome/51.0.2704.103

## Troubleshooting

- [x] I have enabled debug mode: https://pagekit.com/docs/troubleshooting/debug-mode
- [x] I have verified the server requirements: https://pagekit.com/docs/getting-started/requirements
- [ ] I have tried disabling all installed extensions _- only portfolio installed_
- [x] I have checked the browser developer console for JavaScript errors

**Comment by MalteScharenberg** [comment]
Caching should be disabled for MIME Type `text/html` (https://github.com/pagekit/pagekit/blob/develop/.htaccess#L111) .
Can you check if the appropriate headers are set for your page?

**Comment by robertphipps** [comment]
Hi,

I have just checked my .htaccess and it did have that line, but also the line `Options -MultiViews` in the URL rewrites section, so I have replaced it with a copy .htaccess you have linked to, which didn't seem to fix it.

Anyway, I had a fiddle with the Unixy Varnish settings and tried the site in IE11 (still not working). So I reloaded the page a few times and opened IE's developer tools window, and hey presto it seems to have fixed whatever the issue was (on Chrome as well)

So I'm not sure what I'd done, but it now is working fine. Sorry about that, I'm feeling a total noob now.

Cheers,
Rob
